corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -28,6 +28,20 @@ <h1  aria-label="<%- title %>" tabindex="0" class="page-title"><%- title %></h1>
         <div aria-label="<%- description %>" tabindex="0" class="query-description"><%= description %></div>
       <% } %>
     </div>
+    <% if (actions || isMultiSelect) { %>
```

Comment:
can this be made a template?

Reply:
I think so. I'll work on it.

Reply:
@stephherbers  Since we are only concerned with `Continue` action I think it makes sense to remove the template added here https://github.com/dimagi/commcare-hq/pull/33847/commits/5314171df46b2fac10f4ce4aea195e89a5a197f3 and then just add the `Continue` button div on the condition of `isMultiSelect`.

Reply:
I'm also happy to convert that template to be just for the `Continue` button. I was torn as to whether that was too small of a chunk of html to warrant its own template, but there are other button templates.